storage/reads/datatypes/storage_common.proto

```diff
@@ -179,3 +178,55 @@ message TagValuesRequest {
 message StringValuesResponse {
   repeated bytes values = 1;
 }
+
+// MeasurementNamesRequest is the request message for Storage.MeasurementNames.
+message MeasurementNamesRequest {
```

Comment:
I know this isn't an established pattern, but it would be helpful to mention what the response type is for each of these requests.

Reply:
I can add them to the response message:

```protobuf
// Response message for Storage.TagKeys, Storage.TagValues and Storage.MeasurementNames.
message StringValuesResponse {
  repeated bytes values = 1;
}
```

NOTE: I believe our end-game should be to move the majority of these definitions to the service definition, as I don't believe any belong here. It turned out to be a considerable amount of work, so I wanted to do that as a separate task.